tests/test_table_client_aggregation.rs

```diff
@@ -0,0 +1,547 @@
+        .sum("c3".to_owned())
+        .avg("c3".to_owned())
+        .add_scan_range(vec![Value::from(200i32), Value::from(0i64)], true, vec![Value::from(200i32), Value::from(1000i64)], true);
+
+    // get result
+    let result_set = aggregation.execute().await;
+
+    // test
+    let result_set = result_set.unwrap();
+    if !result_set.get("min(c3)".to_owned()).is_none() {
+        assert!(false);
+    }
+    if !result_set.get("max(c3)".to_owned()).is_none() {
+        assert!(false);
+    }
+    if !result_set.get("sum(c3)".to_owned()).is_none() {
+        assert!(false);
+    }
+    if !result_set.get("avg(c3)".to_owned()).is_none() {
+        assert!(false);
+    }
+
+    assert_eq!(0, result_set.get("count(*)".to_owned()).as_i64());
+}
+
+#[tokio::test]
+#[serial]
+async fn test_obtable_client_aggregation_illegal_column() {
+    /*
+         * CREATE TABLE test_partition_aggregation (
+         *  `c1` int NOT NULL,
+         *  `c2` bigint NOT NULL,
+         *  `c3` double DEFAULT NULL,
+         *  `c4` varchar(5) DEFAULT NULL,
+         *  PRIMARY KEY(`c1`, `c2`)) PARTITION BY KEY(`c1`) PARTITIONS 200;
+         */
+
+    let client_handle = task::spawn_blocking(utils::common::build_normal_client);
+    let client = client_handle.await.unwrap();
+    const TEST_TABLE_NAME: &str = "test_partition_aggregation";
+    client.add_row_key_element(TEST_TABLE_NAME, vec!["c1".to_owned()]);
+
+    // clear data
+    client.delete(TEST_TABLE_NAME,
+                  vec![Value::from(200i32), Value::from(50i64)]).await.expect("delete failed");
+    client.delete(TEST_TABLE_NAME,
+                  vec![Value::from(200i32), Value::from(150i64)]).await.expect("delete failed");
+    client.delete(TEST_TABLE_NAME,
+                  vec![Value::from(300i32), Value::from(50i64)]).await.expect("delete failed");
+
+    // prepare data
+    client
+        .execute_sql("insert into test_partition_aggregation(c1, c2, c4) values('200', '50', 'a')").expect("insert fail");
+    client
+        .execute_sql("insert into test_partition_aggregation(c1, c2, c4) values('200', '150', 'a')").expect("insert fail");
+    client
+        .execute_sql("insert into test_partition_aggregation(c1, c2, c4) values('300', '50', 'a')").expect("insert fail");
+
+    // aggregate
+    let aggregation = client
+        .aggregate(TEST_TABLE_NAME)
+        .sum("c4".to_owned())
+        .add_scan_range(vec![Value::from(200i32), Value::from(0i64)], true, vec![Value::from(200i32), Value::from(1000i64)], true);
+
+    // get result
+    let result_set = aggregation.execute().await;
+
+    assert!(result_set.is_err());
+
+    match result_set {
+        Ok(_) => unreachable!(),
+        Err(e) => {
+            assert_eq!(obkv::error::CommonErrCode::ObException(OB_ERR_UNEXPECTED), e.common_err_code().unwrap());
+        }
+    }
+
+    // clear data
+    client.delete(TEST_TABLE_NAME,
+                  vec![Value::from(200i32), Value::from(50i64)]).await.expect("delete failed");
```

Comment:
```suggestion
let result = client.delete(TEST_TABLE_NAME, vec![Value::from(200i32), Value::from(50i64)]).await;
assert!(result.is_ok());
```

Reply:
I guess we should remove all expect in tests and src files

Reply:
fixed